amanuensis/blueprints/project.py

```diff
@@ -12,16 +12,39 @@
 from amanuensis.schema import ProjectSchema
 from cdislogging import get_logger
 
-
 blueprint = flask.Blueprint("projects", __name__)
 
 logger = get_logger(__name__)
 
+def get_state_id_to_code():
+
+    with flask.current_app.db.session as session:
+
+        # Map the State.id to the State.code in case they're in a different order than expected in the DB.
+
+       return {int(state.id): state.code for state in session.query(State).all()}
+
```

Comment:
The blueprints should not contain DB operation. It should be an abstraction so if for any reason we need to change DB or query package in the future it will be applied to the models file only.